### ISSUE TITLE
Broken for custom properties

### ISSUE THREAD

**Issue by clembu** [Mustard2/MenuCreator#5]
The following lines render the property adding incompatible with custom properties:
https://github.com/Mustard2/MenuCreator/blob/9e4a2c06c6743de601addc2da9f352c15a7a513d/menu_creator.py#L402-L403

```python
'["Object"]["prop"]'.rsplit('[', 1)
// outputs [ '["Object"]', '"prop"]' ]
```

the `path` of the Menu property becomes `["Object"]` where it should be `["Object"]["prop"]`

**Comment by Mustard2** [comment]
Thank you, I should look into it.

**Comment by Mustard2** [comment]
I checked the error and indeed it's a problem of how custom properties are implemented in Blender. I added some additional checks and now it should work fine.

Let me know if it's not fixed in the new version (0.0.3), reopening the issue.
